src/greatwall/resources/knowledge/fractal.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         height=None,
         escape_radius=None,
         max_iters=None,
+        computed_values=None
```

Comment:
I agree. I have modified the constructor to not expose the variable outside the class, but kept the initialization of it in the constructor.

Reply:
I agree. I left the variable initialization in the constructor. I should check the PR first myself because this is something I should have seen, thanks.